Pipelines-RN-1.12.2 Release Notes for Pipelines 1.12.2 (#67276)
openshift/openshift-docs: modules/op-release-notes-1-12.adoc

* Before this update, when using {pac}, if you created an access policy on the `Repository` custom resource (CR) that did not include a particular user and then added the user to the `OWNER` file in the Git repository, the user would have no rights for the {pac} CI process. For example, if the user created a pull request into the Git repository. the CI process would not run on this pull request automatically. With this update, a user who is not included in the access policy on the `Repository` CR but is included in the `OWNER` file is allowed to run the CI process for the repository.

* With this update, the HTTP/2.0 protocol is not supported for webhooks. All webhook calls to {pipelines-title} must use the HTTP/1.1 protocol.

[id="release-notes-1-12-2_{context}"]
== Release notes for {pipelines-title} General Availability 1.12.2

With this update, {pipelines-title} General Availability (GA) 1.12.2 is available on {OCP} 4.12 and later versions.

[id="fixed-issues-1-12-2_{context}"]
=== Fixed issues

> * Before this update, the generated git secret for the latest pipeline run would get deleted when the `max-keep-runs` parameter was exceeded. With this update, the git secret no longer gets deleted on the latest pipeline run.

Reviewer: ```suggestion
* Before this update, the generated Git secret for the latest pipeline run was deleted when the `max-keep-runs` parameter was exceeded. With this update, the Git secret is no longer deleted on the latest pipeline run.
```

Per https://redhat-documentation.github.io/supplementary-style-guide/#git Git should be capitalized. Also, suggested editing for tenses.

> * Before this update, the GitHub `basic-auth` secret for the latest pipeline run would get deleted when the `max-keep-runs` parameter was exceeded. With this update, the GitHub `basic-auth` secret no longer gets deleted and it still exists on the cluster if its owner pipeline run also exists on the lcuster.

Reviewer 2: ```suggestion
* Before this update, the generated git secret for the latest pipeline run would get deleted when the `max-keep-runs` parameter was exceeded. With this update, the git secret  no longer gets deleted on the latest PipelineRun
```

it's not specific to Github and it's not a basic-auth secret
